kits/arm/arm.py

```diff
@@ -18,10 +18,71 @@
 
 class ArmParams(object):
     
-    def __init__(self, family, moduleNames, hrdf):
+    def __init__(self, family, moduleNames, hrdf, gripperName=None):
         self.family = family
         self.hrdf = hrdf
         self.moduleNames = moduleNames
+        self.gripperName = gripperName
+        if self.gripperName is not None:
+            self.hasGripper = True
+        else:
+            self.hasGripper = False
```

Comment:
I would probably either take this out or turn it into a function -- relatively minor, but keeping parallel state isn't ideal.

For example, if after the object was constructed without someone later set `params.gripperName = "newName"`, then `hasGripper` would still be `false`, causing runtime errors elsewhere; same with the reverse case here.

Reply:
From my understanding, I think the changes I have made avoid what you've described, even though it requires two pieces of user input when setting up a gripper.